### PR TITLE
Task/eparker71/add manage people and manage sections back

### PR DESCRIPTION
**Pull request by eparker71**

### CODE REVIEW

File: js/left_nav_tool_visibility_indicator.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 var huToolVisibilityRestricted = [
   // Course-level
   "A/B Testing Tool",
+  "Manage People",
+  "Manage Sections",
   "Course Emailer",
   "Import iSites Content",
   "Manage Course",
```
Comment (cmurtaugh) [inline js/left_nav_tool_visibility_indicator.js:9]: Is that trailing comma going to cause problems?

**Comment by elliottyates** [comment]
👍  @cmurtaugh

File: js/left_nav_tool_visibility_indicator.js (hunk 1 above)
Comment (elliottyates) [inline js/left_nav_tool_visibility_indicator.js:9]: Chrome debugger:

```
a=['a',]
["a"]
a=['a','b',]
["a", "b"]
```

Comment (cmurtaugh) [inline js/left_nav_tool_visibility_indicator.js:9]: Cool - just making sure.  Thanks!

**Pull request merged**